core/src/main/java/org/htsjdk/core/exception/HtsjdkException.java

```diff
@@ -0,0 +1,35 @@
+package org.htsjdk.core.exception;
+
+/**
+ * Exception type for all exceptions caused at runtime by HTSJDK-NEXT.
+ */
+public class HtsjdkException extends RuntimeException {
+
+    /**
+     * Constructs an HTSJDK exception without cause.

```

Comment:
I would drop the "without cause" because it's confusing out of the context of the adjacent constructor.

Reply:
Done.